cpp/ql/src/Security/CWE/CWE-428/UnsafeCreateProcessCall.qhelp

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE qhelp PUBLIC
+  "-//Semmle//qhelp//EN"
+  "qhelp.dtd">
+<qhelp>
+
+<overview>
+  <p>This query indicates that there is a call to a function of the <code>CreateProcess*</code> family of functions, which may result in a security vulnerability if the path contains spaces.</p>
```

Comment:
```which may result in a security vulnerability if the path contains spaces```
Perhaps this is a little tentative? Perhaps we should say that the code identified by the query _does_ introduce a vulnerability. Change "which may result in a security vulnerability" to:
```which introduces a security vulnerability```

Reply:
Fixed